android/src/main/java/com/reactlibrary/RNUportHDSignerModule.kt

```diff
@@ -343,4 +344,17 @@ class RNUportHDSignerModule(reactContext: ReactApplicationContext)
         promise.resolve(UportHDSigner().validateMnemonic(phrase ?: ""))
     }
 
+    /**
+     * Returns a list of addresses representing the uport roots used as handles for seeds
+     */
+    @ReactMethod
+    fun allHDRoots(promise: Promise?) {
```

Comment:
The equivalent method on iOS is named `listSeedAddresses`
I suggest using the same name here:

```suggestion
    fun listSeedAddresses(promise: Promise?) {
```